scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -48,12 +54,14 @@ namespace <%= namespace %>.SDK {
         public string ID = null;
         private string _VariantId;
         private long _Quantity;
+        private decimal _Price;
         private ObservableDictionary<string, string> _CustomAttributes;
         private OnCartLineItemChange OnChange;
-        
-        public CartLineItem(string variantId, OnCartLineItemChange onChange, long quantity = 1, IDictionary<string, string> customAttributes = null) {
+
+        public CartLineItem(string variantId, OnCartLineItemChange onChange, decimal price, long quantity = 1, IDictionary<string, string> customAttributes = null) {
```

Comment:
Same thing goes with removing `AddOrUpdate(string variantId, decimal price ...)` since we have `AddOrUpdate(ProductVariant variant, ...`